model/src/main/java/org/cbioportal/model/StructuralVariant.java

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.cbioportal.model;
+
+import java.io.Serializable;
+import org.cbioportal.model.UniqueKeyBase;
+
+public class StructuralVariant extends UniqueKeyBase implements Serializable{
+
+    private String molecularProfileId;
+    private long structuralVariantId;
+    private int sampleIdInternal;
```

Comment:
same comment as before regarding storing the sample stable id and internal sample id in data model